backend/app/emails.py

```diff
@@ -46,6 +46,7 @@ class EmailInfo(BaseModel):
     template_data: dict[str, Any]
     # Only works when app_id is set and email is not user specific
     inform_moderators: bool = False
+    inform_only_moderators: bool = False
```

Comment:
Do we really want the four cases this generates in combination with `inform_moderators`

I would think we only want three?
- Users
- Moderators
- All

would  `inform_moderators = false` and `inform_only_moderators = true` even make sense?

Reply:
We should probably rework this to always send specific types of messages to moderators directly in the e-mail function but I will be worrying about this later